src/schema.rs

```diff
@@ -58,7 +58,7 @@ impl Schema {
     /// instance plus the provided type.  Note that the added type
     /// in the returned instance will hide a type of the same name
     /// from this instance.
-    fn plus_type(&self, schema_type: TypeDefinition) -> Self {
+    fn plus_type(&self, schema_type: NamedTypeDefinition) -> Self {
```

Comment:
question—apologies if I'm getting the models confused, but should this be `IslType` (the model for constructing new types)?

Reply:
Yes. That is correct. I will change this to have `IslType` in the method parameter.